core/templates/services/site-analytics.service.ts

```diff
@@ -36,6 +36,8 @@ import {NavbarAndFooterGATrackingPages} from 'app.constants';
 })
 export class SiteAnalyticsService {
   static googleAnalyticsIsInitialized: boolean = false;
+  private diagnosticTestsStarted: number = 0;
```

Comment:
Do we need these variables?
If not, please remove it from here & from the test file.